### PR TITLE
Feature: switch diagram type

### LINKED ISSUE #281: Project page (index page): switch display view of diagrams (list <-> grid)
## Description

On the project page (index page), add buttons to switch the diagrams display type view.

Both buttons are in primary color.
Only one button is selected, selected button is disabled.

If "list" button is selected display the diagrams list.
If "grid" button is selected display the diagrams grid.

Save in localstorage the state of the button.

## Mockup

image

### PR DESCRIPTION
**Pull request by justinehell**
Resolve #281

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -36,6 +36,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Multi diagrams page:
   * Display all diagrams of a project.
   * Zoom centered on all diagrams.
+* Index page:
```
Comment (Zorin95670) [inline changelog.md:39]: Wrong place

Comment (justinehell) [reply changelog.md]: done

File: tests/unit/components/ModelsView.spec.js [hunk 2]
```diff
@@ -73,15 +73,33 @@ describe('Test component: ModelsView', () => {
     });
   });
 
-  describe('Test props: projectName', () => {
+  describe('Test prop: projectName', () => {
```
Comment (k-y0u) [inline tests/unit/components/ModelsView.spec.js:76]: no need this update

File: cypress/e2e/ModelsPage/SwitchDiagramType.feature [hunk 3]
```diff
@@ -0,0 +1,33 @@
+Feature: Test models page: switch from table to grid type
```
Comment (k-y0u) [inline cypress/e2e/ModelsPage/SwitchDiagramType.feature:1]: grid display type

File: tests/unit/components/ModelsView.spec.js (hunk 2 above)
Comment (justinehell) [reply tests/unit/components/ModelsView.spec.js:76]: done

File: cypress/e2e/ModelsPage/SwitchDiagramType.feature (hunk 3 above)
Comment (justinehell) [reply cypress/e2e/ModelsPage/SwitchDiagramType.feature:1]: done

**Review by k-y0u — APPROVED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**